nais/vars-q1.yaml

```diff
@@ -10,6 +10,8 @@ abac_pdp_endpoint_url: https://wasapp-q1.adeo.no/asm-pdp/authorize
 norg_url: https://app-q1.adeo.no/norg2/ws/OrganisasjonEnhet/v2
 oppgave_url: https://oppgave.nais.preprod.local/api/v1
 isso_isalive_url: https://isso-q.adeo.no/isso/isAlive.jsp
+kafka_url: b27apvl00045.preprod.local:8443,b27apvl00046.preprod.local:8443,b27apvl00047.preprod.local:8443
```

Comment:
@chriswk - kan du gjøre en QA av spesielt disse to verdiene? Vi hadde ikke lagt til noe for produksjon, og fant noe i et annet repo, men vet ikke om det er riktig.